examples/src/main/java/io/dapr/examples/secrets/README.md

```diff
@@ -46,167 +43,93 @@ Then get into the examples directory:
+### Creating a JSON secret file locally
 
-1. To run the vault container locally run: 
-<!-- Docker is writing output to stderr ... -->
+Dapr's API for secret store only support read operations. For this sample to run, we will first create a secret file with a JSON string that contains two keys: `redisPassword` and `randomKey`.
 
 <!-- STEP
-name: Start vault
-expected_stderr_lines:
-  - 'Creating network "secrets_default" with the default driver'
-sleep: 10 
+name: create local file 
 -->
 
 ```bash
-docker-compose -f ./src/main/java/io/dapr/examples/secrets/docker-compose-vault.yml up -d
-``` 
-
-<!-- END_STEP -->
-
-2. Run `docker ps` to see the container running locally: 
-
-```bash
-342d3522ca14        vault                      "docker-entrypoint.s…"         34 seconds ago        Up About
-a minute   0.0.0.0:8200->8200/tcp                               secrets_hashicorp_vault_1
-```
-Click [here](https://hub.docker.com/_/vault/) for more information about the container image for Hashicorp's Vault.
-
-### Create a secret in Vault
-Dapr's API for secret store only support read operations. For this sample to run, we will first create a secret via the Vault's cli commands:
-
-Export the `VAULT_ADDR` for vault CLI:
-```bash
-export VAULT_ADDR=http://127.0.0.1:8200/
-```
-
-Login to Hashicorp's Vault:
-
-<!-- STEP
-name: Vault login
-expected_stdout_lines:
-  - "Success! You are now authenticated. The token information displayed below"
-  - "token                myroot"
-env:
-  VAULT_ADDR: "http://127.0.0.1:8200/"
--->
-
-```bash
-vault login myroot
+echo '{"redisPassword":"root123","randomKey":"value"}' > ./components/secrets/secret.json
 ```
 
 <!-- END_STEP -->
 
-Create secret (replace `$MY_FAVORITE_MOVIE` with a title of our choice):
-
-<!-- STEP
-name: Create movie vault secret
-expected_stdout_lines:
-  - "version            1"
-env:
-  VAULT_ADDR: "http://127.0.0.1:8200/"
-  MY_FAVORITE_MOVIE: "Star Wars"
--->
-
-```bash
-vault kv put secret/dapr/movie title="$MY_FAVORITE_MOVIE"
-```
-
-<!-- END_STEP -->
-
-Create random secret:
-
-<!-- STEP
-name: Create random vault secret
-expected_stdout_lines:
-  - "version            1"
-env:
-  VAULT_ADDR: "http://127.0.0.1:8200/"
--->
-
-```bash
-vault kv put secret/dapr/randomKey testVal="value"
-```
-
-<!-- END_STEP -->
-
-In the command above, `secret` means the secret engine in Hashicorp's Vault.
-Then, `dapr` is the prefix as defined in `< repo dir >/examples/components/hashicorp_vault.yaml`.
-Finally, `movie` and `randomKey` are the secret names with the value set in the form of `key=value` pair.
-
-A secret in Dapr is a dictionary.
-
 ### Running the secret store sample
 
 The example's main function is in `SecretClient.java`.
 
 ```java
 public class SecretClient {
+
     /**
      * Identifier in Dapr for the secret store.
      */
-    private static final String SECRET_STORE_NAME = "vault";
-  
+    private static final String SECRET_STORE_NAME = "local-secret-store";
```

Comment:
I think this is an opportunity to keep this name generic, so users can understand that this can be any vault.